### PR TITLE
Install guide

### PR DESCRIPTION
**Pull request by keetron**
Made an easy step by step guide on how to install FitNesse using IntelliJ / Maven.
The rest of the tutorials will be in .wiki format but to be able to run the wiki you need an installation, hence this.

### CODE REVIEW

**Comment by keetron** [comment]
Shit, some rogue commit got in somehow, let me clean up

File: InstallGuide.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+To run FitNesse, simply click the little green triangle next to the `Run Configuration` button:
+
+![FitNesse Run Button](wiki/FitNesseRoot/Manual/img/RunFitNesse.PNG)
+
+In the bottom of the IntelliJ window, a console should come up and start doing things, FitNesse could throw a firewall warning, just accept this as some things need to be pulled from internet repositories. When it done, it should look somewhat like this:
+
+![FitNesse Running](wiki/FitNesseRoot/Manual/img/FitNesseRunning.PNG)
+
+
+Now use your preferred browser to navigate to <http://localhost:9090/> where your FitNesse installation is now up and running. Instructions on how to write tests can be found from the frontpage.
```
Comment (fhoeben) [inline InstallGuide.md]: Can 'frontpage' be a link to somewhere?

File: InstallGuide.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * Intellij (including the FitNesse Plugin) to edit the test files
+ 
+#####Git
+ Go to <https://git-scm.com/downloads>, download and install Git using the default settings.
+#####Java JDK
+On <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html> select the correct JDK for windows, do not forget to click *Accept Licence* first. After download, install this one as well.
+#####IntelliJ
+Download IntelliJ from <https://www.jetbrains.com/idea/download/>, specifically choose the Community Edition. After download, install this as well using the default options.
+######Chrome
+While it can be assumed most people have this already, just to make you do too.
```
Comment (fhoeben) [inline InstallGuide.md]: Other products have a link to where they can be obtained. I would say include a link to chrome download also.

File: InstallGuide.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# FitNesse / HSAC installation guide
+
+This is a step by step guide to get HSAC up and running on your machine. This guide is using Windows 10 as an OS, the steps will be similar in another environment. This guide uses IntelliJ as an Integrated Development Environment as it will allow you to get most out of FitNesse and to manage the running and updating of FitNesse as well as version control on the test files you are creating.
+ 
+ ##Supporting tools setup
+ We need to install these tools:
+ * Git for version control
+ * Java JDK so we can build the FitNesse framework from code
+ * Intellij (including the FitNesse Plugin) to edit the test files
```
Comment (fhoeben) [inline InstallGuide.md]: Should chrome be in this list?

Comment (fhoeben) [inline InstallGuide.md]: I believe you want to add instructions on how to install FitNesse Plugin also.

**Comment by fhoeben** [comment]
Should the readme.md be updated to link to the install guide?
Or can we place the install guide in the [wiki section of this project (in GitHub)](https://github.com/fhoeben/sample-fitnesse-project/wiki) instead of including in the project source (once users are up and running there is little added value to have it, right), and link to the page there?

File: InstallGuide.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+On <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html> select the correct JDK for windows, do not forget to click *Accept Licence* first. After download, install this one as well.
+#####IntelliJ
+Download IntelliJ from <https://www.jetbrains.com/idea/download/>, specifically choose the Community Edition. After download, install this as well using the default options.
+######Chrome
+While it can be assumed most people have this already, just to make you do too.
+ 
+##FitNesse project setup
+With the above tooling in place, create a folder on a local drive where the FitNesse project will be placed.
+
+Open IntelliJ, just click Next a lot and a window should appear that will look something like this:
```
Comment (fhoeben) [inline InstallGuide.md]: What is the name of the program (in the start menu) to be launched in Windows, to you 'Open IntelliJ' is clear, but will it be to users that never used it?

File: InstallGuide.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+##Build and Run FitNesse
+
+#####Build FitNesse
+Now we will build and start up FitNesse, this is the final step before we can start writing tests. For this we need to add a Run Configuration, use the button in the screenshot below and click `Edit Configurations`.
+
+![EditConfigurations](wiki/FitNesseRoot/Manual/img/RunConfiguration.PNG)
+
+Press the `+` button to add a configuration and select `Maven`.
+
+![Maven](wiki/FitNesseRoot/Manual/img/Maven.PNG)
```
Comment (fhoeben) [inline InstallGuide.md]: Does this just work in IntelliJ, or should users also download and unzip Maven, setup environment variables (like JAVA_HOME and M2_HOME)?

File: InstallGuide.md (hunk 3 above)
Comment (keetron) [inline InstallGuide.md]: I forgot the plugin, it was on my mental list. Will add that.

File: InstallGuide.md (hunk 2 above)
Comment (keetron) [inline InstallGuide.md]: For consistency, I agree.

File: InstallGuide.md (hunk 4 above)
Comment (keetron) [inline InstallGuide.md]: Well, after installing it there is an option to run it right away and I have to trust the users they can navigate the start menu and other rather basic Windows stuff such as opening an application they just installed.

File: InstallGuide.md (hunk 5 above)
Comment (keetron) [inline InstallGuide.md]: Nope, this worked out of the box. I was almost as surprised as you are but I ran it all on a fresh Win10 Virtualbox VM to check if these instructions made sense and to ensure I forgot nothing.

File: InstallGuide.md (hunk 1 above)
Comment (keetron) [inline InstallGuide.md]: After navigating to that link, you end up on the FrontPage, where a menu is where to go next. Adjusting the FrontPage is a US to be implemented 😄

**Comment by keetron** [comment]
There are 5 comments saying the same?
Either way, you are right in that this install instruction should be accessible before the code is accessed. Didn't think that one through, now did I? Let me sniff around for a good place or suggest one and I will make the changes.

**Comment by keetron** [comment]
Ok, added a wiki page, it is now at the bottom of the little menu and now sure how to change that.
The pictures are linked to the branch I used to make this PR in my fork, not very pretty, it is very effective.
I will adjust the PR to include only the pictures and after you merged, change the wiki page.

**Comment by keetron** [comment]
Wait, wrong button.

**Pull request closed without merging**